test/index.test.ts

```diff
@@ -68,6 +68,12 @@ test('Should ensure formatting is correct for PublicInputs', async function (ass
 test('Should ensure formatting is correct for extracted PublicInputs', async function (assert) {
   // For each circuit
   testVectors.forEach((testVector, i) => {
+    // Ensure testVector proof exists
+    if (!testVector.proof) {
+      console.log('No proof found for test vector', i)
```

Comment:
Should this fail the test? Alternatively, instead of relying on `console` in the tests, there's `assert.comment` explicitly for this purpose. Then the test output conforms to the [TAP standard](https://testanything.org)